blocks/footer/footer.css

```diff
@@ -11,3 +12,131 @@ footer .footer {
 footer .footer p {
   margin: 0;
 }
+
+/* container for footer nav and copyright */
+footer .footer > div {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+/* flex container for footer nav */
+footer .columns > div {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  gap: 10px;
+}
+
+/* two columns for mobile */
+@media (max-width: 768px) {
```

Comment:
it should be the other way round. mobile styles are default, then override for larger resolutions with `min-width` instead.